admin/client.py

```diff
@@ -116,14 +116,25 @@ class DockerRunner:
     """
 
     def __init__(self, image):
-        self.docker = docker.Client(version='1.18')
+        # Getting Docker to work correctly on any client platform can
+        # be tricky.  See
+        # http://doc-dev.clusterhq.com/gettinginvolved/client-testing.html
+        # for details.
+        params = docker.utils.kwargs_from_env(assert_hostname=False)
```

Comment:
OK, `verify=False` is usually a bad thing to do, so I will remove that, and keep the `assert_hostname` change.